.github/workflows/code-coverage.yml

```diff
@@ -25,16 +25,16 @@ jobs:
         fetch-depth: 0 # fetching all
 
     - name: Install dependencies
-      run: dotnet restore
+      run: dotnet restore OpenTelemetry.sln
 
     - name: dotnet build
-      run: dotnet build --configuration Release --no-restore
+      run: dotnet build OpenTelemetry.sln --configuration Release --no-restore
```

Comment:
Yes, that would have required you to specify the .sln file when using `dotnet build`. I have moved `InstrumentationLibraries.proj` to a `build` to avoid that.